pkg/util/testutil/kourier.go

```diff
@@ -0,0 +1,43 @@
+package testutil
+
+import (
+	"context"
+	"testing"
+
+	"github.com/puppetlabs/leg/k8sutil/pkg/manifest"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func doInstallKourier(ctx context.Context, cl client.Client, mapper meta.RESTMapper) error {
+	// Kourier requires Knative Serving; it won't detect it after the fact.
```

Comment:
I think this may only be true of Ambassador FWIW.

Reply:
Yeah, I figured both of those comments were probably no longer applicable, but I wasn't certain enough to remove them (yet). I don't think either comment are really necessary at this point, regardless, though, so I'll either remove them now or in the next update.